package.json

```diff
@@ -1,44 +1,48 @@
 {
-  "name": "@moxy/react-lib-template",
+  "name": "@moxy/react-contentful-image",
   "version": "0.0.0",
-  "description": "A template that aims to make the implementation of React component packages easier and more methodic.",
+  "description": "A react image rendering using Contentful Images API.",
```

Comment:
Change (or not) according to: https://github.com/moxystudio/react-contentful-image/pull/1/files#r416639998